webapp/tokenList/stakeTokens.ts

```diff
@@ -95,6 +95,13 @@ export const stakeWhiteList: Partial<
       rewards: ['hemi'],
       website: 'https://kerneldao.com/kelp',
     },
+    // satUSD
+    '0xb4818BB69478730EF4e33Cc068dD94278e2766cB': {
+      protocol: 'satoshi',
+      priceSymbol: 'usdc',
+      rewards: ['hemi3x', 'satoshi', 'pell'],
+      website: 'https://www.satoshiprotocol.org',
+    },
```

Comment:
just to confirm, did max whitelisted the token?

Reply:
Token whitelisted